### DISCUSSION TITLE
Why isn't Scratch Addons updating to 1.18.0?

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#3231] [Help] [answered]
On Firefox whenever I try check for Scratch Addons updates, it says "No updates found." even though 1.18 was released over 3 hours ago. The only unusual thing I found is that this warning pops up in the addon debugger after checking for updates, does that have something to do with it?
![image](https://user-images.githubusercontent.com/81489795/128075387-5eb09fb9-7169-4e00-be6b-bde94151a0ef.png)

**Comment by CST1229** [comment, marked as answer]
It needs to be sent for review then reviewed (on the Discord W_L said that it will be sent after Chrome finishes review)

**Reply by Samq64** [reply]
That's strange, beacuse Firefox usually receives the update immediately after it's released.

**Reply by WorldLanguages** [reply]
I changed the strategy, I now upload on Firefox after Chrome accepts the review so that they're released at the same time.

**Reply by Samq64** [reply]
~~That's too bad, because I liked getting the updates before everyone else.~~

OK, thanks!

**Reply by WorldLanguages** [reply]
Yeah, the fact there's now a changelog video released in YouTube changed things. I don't want firefox users to have a broken YouTube link, and I don't want Chrome users to watch something they can't get yet.

**Reply by Samq64** [reply]
> Yeah, the fact there's now a changelog video released in YouTube changed things. I don't want firefox users to have a broken YouTube link, and I don't want Chrome users to watch something they can't get yet.

I guess you're right. I'll just have to wait a couple of hours to update.

**Reply by WorldLanguages** [reply]
Well, Chrome review took too long so I ended up making the video unlisted, released in Firefox, then released in Chrome this morning, after making the video public.

**Reply by Secret-chest** [reply]
> Yeah, the fact there's now a changelog video released in YouTube changed things. I don't want firefox users to have a broken YouTube link, and I don't want Chrome users to watch something they can't get yet.

Yeah, it would be unfair. I use plain Chromium, google chrome UI but opensource
